capplets/appearance/appearance-font.c

```diff
@@ -781,6 +802,40 @@ void font_init(AppearanceData* data)
 			  G_CALLBACK (marco_changed),
 			  data);
 
+	/*In a wayland session we must manage MATE font settings for xwayland
+	 *and also manage GNOME font settings for native wayland applications
+	 *
+	 *First find out if we are running under a wayland session
+	 */
+
+	ret = system ("killall -0 -e Xwayland");
```

Comment:
That's a bit… meh.  Can't we just check whether the app's display is a Wayland one?  The only probable issue is if the app is run under Xwayland, but maybe it's only fair that it might not control the Wayland session in that case?

Reply:
The original reason for this was that at first I had this whole capplet running under xwayland. Once I got past that and got it running under native wayland, there was no more need for this roundabout approach. One of today's updates removed the leftover instance of this,